src/main/steps/c100-rebuild/respondent-details/util.ts

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { v4 as uuidv4 } from 'uuid';
+
+import { C100RebuildPartyDetails, ChildrenDetails, Gender, YesNoEmpty } from '../../../app/case/definition';
+
+export const getDataShape = (): C100RebuildPartyDetails => ({
+  id: uuidv4(),
+  firstName: '',
+  lastName: '',
+  personalDetails: {
+    dateOfBirth: {
+      day: '',
+      month: '',
+      year: '',
+    },
+    isDateOfBirthUnknown: YesNoEmpty.EMPTY,
+    approxDateOfBirth: {
+      day: '',
+      month: '',
+      year: '',
+    },
+    gender: Gender.EMPTY,
+    otherGenderDetails: '',
+  },
+  relationshipDetails: {
+    relationshipToChildren: [
+      {
+        relationshipType: '',
+        childId: '',
+        otherRelationshipTypeDetails: '',
+      },
+    ],
+  },
+});
+
+export const getRespndentDetails = (
```

Comment:
We need to move these to one common util inside c100-rebuild so all can access these methods.